AUTHORS.md

```diff
@@ -1,105 +1,141 @@
 TypeScript is authored by:
-
+* Abubaker Bashir
 * Adam Freidin
-* Ahmad Farid 
-* Akshar Patel

```

Comment:
Is it just me or did this commit (accidentally?) remove some contributors from the list?